package.json

```diff
@@ -54,6 +54,7 @@
     "@babel/code-frame": "7.24.2",
     "@gemini-testing/commander": "2.15.4",
     "@jspm/core": "2.0.1",
+    "@puppeteer/browsers": "2.4.0",
```

Comment:
- download chrome
- download chromium (replaces chrome for versions >73, <115)
- download firefox